packages/sit-onyx/src/components/OnyxPageLayout/OnyxPageLayout.vue

```diff
@@ -17,31 +17,27 @@ const slots = defineSlots<{
 
 const pageModifier = computed(() => {
   let mode = "";
-  if (!slots.footer && slots.sidebar) mode = "onyx-page--side-main";
-  if (slots.footer && (!slots.sidebar || props.hideSidebar)) mode = "onyx-page--main-footer";
-  if (slots.footer && slots.sidebar) {
-    if (props.footerAsideSidebar) {
-      mode = "onyx-page--side-main-footer-partial";
-    } else {
-      mode = "onyx-page--side-main-footer-full";
-    }
+  if (props.footerAsideSidebar) {
+    mode = "onyx-page--footer-partial";
+  } else {
+    mode = "onyx-page--footer-full";
   }
   return mode;
 });
 </script>
 
 <template>
   <div class="onyx-page" :class="pageModifier">
-    <aside v-if="slots.sidebar && !props.hideSidebar" class="onyx-page__sidebar">
+    <aside v-show="slots.sidebar && !props.hideSidebar" class="onyx-page__sidebar">
       <slot name="sidebar"></slot>
     </aside>
     <main class="onyx-page__main">
       <slot></slot>
     </main>
-    <footer v-if="slots.footer" class="onyx-page__footer">
+    <footer v-show="slots.footer" class="onyx-page__footer">
       <slot name="footer"></slot>
     </footer>
-    <div v-if="slots.toasts" class="onyx-page__toasts">
+    <div v-show="slots.toasts" class="onyx-page__toasts">
```

Comment:
Toast will be displayed fixed using the popover API, right so I think this slot is obsolete :)

Reply:
I thought about that but as soon as we get the bottom bar we might need to rethink the approach?